### PR TITLE
Experiment with passive private following in chat

### PR DESCRIPTION
**Pull request by mlsteele**

### CODE REVIEW

**Comment by maxtaco** [comment]
Ok, this looks cool. Also need a way to send down a `TrackUV` from the front-end when the user clicks "let them back in"

File: go/libkb/pegboard.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	if !p.enabled {
+		return nil
+	}
+	defer mctx.Trace("Pegboard.TrackUPAK", func() error { return err })()
+	p.mu.Lock()
+	defer p.mu.Unlock()
+	if peg, ok := p.store[upak.Uid]; ok {
+		// User is already tracked.
+		// CORE-10522 check with explicit local and remote follows?
+		if peg.EldestSeqno != 0 && upak.EldestSeqno != 0 && upak.EldestSeqno < peg.EldestSeqno {
```
Comment (maxtaco) [inline go/libkb/pegboard.go]: combine two `if`s into an `&&`

File: go/libkb/pegboard.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+			return fmt.Errorf("Cannot older version of user: %v %v < %v", upak.Uid, upak.EldestSeqno, peg.EldestSeqno)
```
Comment (maxtaco) [inline go/libkb/pegboard.go]: missing word in error

File: go/teams/service_helper.go [hunk 3]
```diff
@@ -433,6 +433,7 @@ func reAddMemberAfterResetInner(ctx context.Context, g *libkb.GlobalContext, tea
 	if err != nil {
 		return err
 	}
+	_ = g.Pegboard.TrackUPAK(libkb.NewMetaContext(ctx, g), upak.Current)
```
Comment (mlsteele) [inline go/teams/service_helper.go]: > Also need a way to send down a TrackUV from the front-end when the user clicks "let them back in"
This does that. Frontend calls `ReAddMemberAfterReset` when "let them back in" is tapped. May want to split RPCs or change the name in the future depending on how different the UI is.

**Comment by mlsteele** [comment]
ready again

**Pull request merged**